install/kubernetes/cilium/charts/hubble-cli/templates/daemonset.yaml

```diff
@@ -0,0 +1,60 @@
+{{- if .Values.global.hubble.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: hubble-cli
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: hubble-cli
+spec:
+  selector:
+    matchLabels:
+      k8s-app: hubble-cli
+  template:
+    metadata:
+      labels:
+        k8s-app: hubble-cli
+    spec:
+{{- if .Values.global.affinity }}
+      affinity:
+{{ toYaml .Values.global.affinity | indent 8 }}
+{{- end }}
+      containers:
+        - name: hubble-cli
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - "hubble"
+          args:
+{{- toYaml .Values.args | trim | nindent 10 }}
+          env:
+          - name: HUBBLE_DEFAULT_SOCKET_PATH
```

Comment:
This requires https://github.com/cilium/hubble/pull/239 to be merged, right?

Reply:
Not for `v0.5.1`!

Reply:
Yup, I realized down there that you're using `v0.5.1`. So do we want to stick to latest release for the CI tests? Shouldn't we also run latest master to catch eventual regressions (not as part of this PR though)?